src/components/panel/panel.js

```diff
@@ -743,6 +743,12 @@ function MdPanelService($rootElement, $rootScope, $injector, $window) {
    * @type {enum}
    */
   this.yPosition = MdPanelPosition.yPosition;
+
+  /**
+   * Margin between the edges of the panel and the viewport
+   * @type {number}
+   */
+  this.viewportMargin = MdPanelPosition.viewportMargin;
```

Comment:
I don't think this one needs to be re-exported via `$mdPanel`, since there are no values for people to access or change.